staging/src/k8s.io/apimachinery/pkg/api/resource/quantity.go

```diff
@@ -697,7 +697,7 @@ func (q *Quantity) MilliValue() int64 {
 	return q.ScaledValue(Milli)
 }
 
-// ScaledValue returns the value of ceil(q * 10^scale); this could overflow an int64.
+// ScaledValue returns the value of ceil(q / 10^scale); this could overflow an int64.
```

Comment:
The new comment is accurate, thanks.

An example in the comment to help with intuition might be useful, e.g.:
```
For example, `NewQuantity(1, DecimalSI).ScaledValue(Milli)` returns 1000.
```